tests/testthat/test-githubbio-packages.R

```diff
@@ -0,0 +1,54 @@
+# Run this test for users to make sure the long running job feature
+# of doAzureParallel are still working
+context("long running job scenario test")
```

Comment:
context should be "installing github package test"

Reply:
fixed.